helm/korifi/values.schema.json

```diff
@@ -182,9 +182,13 @@
                 "diskMB": {
                   "description": "Disk in MB for staging.",
                   "type": "integer"
+                },
+                "imageCacheMB": {
+                  "description": "Persistent disk in MB for image caching",
+                  "type": "integer"
                 }
               },
-              "required": ["memoryMB", "diskMB"]
+              "required": ["memoryMB", "diskMB", "imageCacheMB"]
```

Comment:
Is this supposed to be optional? Its defaulted to 2GB if not set by kpack right? https://github.com/pivotal/kpack/blob/d0bc8bd6b5cab0d581240e34784f1887ce01d652/pkg/apis/build/v1alpha2/image_validation.go#L33

Reply:
I see that we are setting it to a default value of `2048` in `values.yaml` if user doesn't specify one. This kind of makes it optional for the user. But should we be even setting this values if user doesn't specify it?

Reply:
I was wondering that myself, but we don't appear to have any optional values in the helm chart, so I made it required to follow the conventions we seem to have in place.

Reply:
I think setting our own default here and requiring it makes it more apparent to the operator what the default actually is. You have to have some pretty deep knowledge of kpack to know it will create a 2GB volume by default.

I also think it's worth calling out that we should treat these as general staging parameters for Korifi since technically you could replace the builder implementation with something other than kpack which may have different defaults or not support build artifact caching at all (this may be an argument for making it optional, but I'd say that's something we could do later).